### PR TITLE
Waxholm rat atlas instances

### PR DESCRIPTION
**Pull request by UlrikeS91**
Added:
- brainAtlas 
- brainAtlasVersions: v1.01, v2, v3, v3.01 & v4
- commonCoordinateSpace (v1.01)

### CODE REVIEW

File: instances/atlas/brainAtlas/WHSSD_atlas.jsonld [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        {
+            "@id": "https://openminds.ebrains.eu/instances/brainAtlasVersion/WHSSD_atlas_v3.01"
+        },
+        {
+            "@id": "https://openminds.ebrains.eu/instances/brainAtlasVersion/WHSSD_atlas_v4"  
+        }
+    ],
+    "homepage": null,
+    "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the atlas version(s) you have used.",
+    "shortName": "WHS rat brain atlas"
```
Comment (lzehl) [inline instances/atlas/brainAtlas/WHSSD_atlas.jsonld]: I suggest to use: WHSSD atlas
(we do not know if in future there will be another WHS rat atlas not Sprague Dawley)

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v1-01.jsonld [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            "@id": "https://openminds.ebrains.eu/instances/licenses/ccBySa4.0"
+        }
+    ],
+    "ontologyIdentifier": null,
+    "otherContribution": null,
+    "relatedPublication": null,
+    "releaseDate": "2014-07-17",
+    "repository": null,
+    "shortName": "WHS rat brain atlas v1.01",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v1-01.jsonld]: cf. comment above: WHSSD atlas v1.01

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v2.jsonld [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            "@id": "https://openminds.ebrains.eu/instances/licenses/ccBySa4.0"
+        }
+    ],
+    "ontologyIdentifier": null,
+    "otherContribution": null,
+    "relatedPublication": null,
+    "releaseDate": "2015-02-02",
+    "repository": null,
+    "shortName": "WHS rat brain atlas v2",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v2.jsonld]: cf. comment above: WHSSD atlas v2

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v3-01.jsonld [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            "@id": "https://openminds.ebrains.eu/instances/licenses/ccBySa4.0"
+        }
+    ],
+    "ontologyIdentifier": null,
+    "otherContribution": null,
+    "relatedPublication": null,
+    "releaseDate": "2021-10-01",
+    "repository": null,
+    "shortName": "WHS rat brain atlas v3.01",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v3-01.jsonld]: cf. comment above: WHSSD atlas v3.01

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v3.jsonld [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            "@id": "https://openminds.ebrains.eu/instances/licenses/ccBySa4.0"
+        }
+    ],
+    "ontologyIdentifier": null,
+    "otherContribution": null,
+    "relatedPublication": null,
+    "releaseDate": "2019-05-15",
+    "repository": null,
+    "shortName": "WHS rat brain atlas v3",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v3.jsonld]: cf. comment above: WHSSD atlas v3

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v4.jsonld [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            "@id": "https://openminds.ebrains.eu/instances/licenses/ccBySa4.0"
+        }
+    ],
+    "ontologyIdentifier": null,
+    "otherContribution": null,
+    "relatedPublication": null,
+    "releaseDate": "2021-10-01",
+    "repository": null,
+    "shortName": "WHS rat brain atlas v4",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v4.jsonld]: cf. comment above: WHSSD atlas v4

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v1-01.jsonld [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            "@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpace/WHSSD_v1.01"
+        }
+    ],
+    "copyright": null,
+    "custodian": null,
+    "description": null,
+    "digitalIdentifier": null,
+    "fullDocumentation": null,
+    "fullName": "Waxholm Space atlas of the Sprague Dawley rat brain v1.01",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v1-01.jsonld]: delete version id

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v1-01.jsonld [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+    "funding": null,
+    "hasTerminologyVersion": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationTerminologyVersion/WHSSD_terminology_v1.01"
+        }
+    ],
+    "homepage": null,
+    "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see "Related publications" below).",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v1-01.jsonld]: double quotes within a double quoted string need to be single quotes -> "aaa 'bbb' ccc"

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v2.jsonld [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    "fullDocumentation": null,
+    "fullName": "Waxholm Space atlas of the Sprague Dawley rat brain v2",
+    "funding": null,
+    "hasTerminologyVersion": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationTerminologyVersion/WHSSD_terminology_v2"
+        }
+    ],
+    "homepage": null,
+    "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see "Related publications" below).",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v2.jsonld]: double quotes within a double quoted string need to be single quotes -> "aaa 'bbb' ccc"

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v3-01.jsonld [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+    "fullDocumentation": null,
+    "fullName": "Waxholm Space atlas of the Sprague Dawley rat brain v3.01",
+    "funding": null,
+    "hasTerminologyVersion": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationTerminologyVersion/WHSSD_terminology_v3.01"
+        }
+    ],
+    "homepage": null,
+    "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see "Related publications" below).",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v3-01.jsonld]: double quotes within a double quoted string need to be single quotes -> "aaa 'bbb' ccc"

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v3.jsonld [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+    "fullDocumentation": null,
+    "fullName": "Waxholm Space atlas of the Sprague Dawley rat brain v3",
+    "funding": null,
+    "hasTerminologyVersion": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationTerminologyVersion/WHSSD_terminology_v3"
+        }
+    ],
+    "homepage": null,
+    "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see "Related publications" below).",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v3.jsonld]: double quotes within a double quoted string need to be single quotes -> "aaa 'bbb' ccc"

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v4.jsonld [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+    "fullDocumentation": null,
+    "fullName": "Waxholm Space atlas of the Sprague Dawley rat brain v4",
+    "funding": null,
+    "hasTerminologyVersion": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationTerminologyVersion/WHSSD_terminology_v4"
+        }
+    ],
+    "homepage": null,
+    "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see "Related publications" below).",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v4.jsonld]: double quotes within a double quoted string need to be single quotes -> "aaa 'bbb' ccc"

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v2.jsonld [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            "@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpace/WHSSD_v1.01"
+        }
+    ],
+    "copyright": null,
+    "custodian": null,
+    "description": null,
+    "digitalIdentifier": null,
+    "fullDocumentation": null,
+    "fullName": "Waxholm Space atlas of the Sprague Dawley rat brain v2",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v2.jsonld]: delete version id

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v3-01.jsonld [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            "@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpace/WHSSD_v1.01"
+        }
+    ],
+    "copyright": null,
+    "custodian": null,
+    "description": null,
+    "digitalIdentifier": null,
+    "fullDocumentation": null,
+    "fullName": "Waxholm Space atlas of the Sprague Dawley rat brain v3.01",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v3-01.jsonld]: delete version id

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v3.jsonld [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            "@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpace/WHSSD_v1.01"
+        }
+    ],
+    "copyright": null,
+    "custodian": null,
+    "description": null,
+    "digitalIdentifier": null,
+    "fullDocumentation": null,
+    "fullName": "Waxholm Space atlas of the Sprague Dawley rat brain v3",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v3.jsonld]: delete version id

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v4.jsonld [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            "@id": "https://openminds.ebrains.eu/instances/commonCoordinateSpace/WHSSD_v1.01"
+        }
+    ],
+    "copyright": null,
+    "custodian": null,
+    "description": null,
+    "digitalIdentifier": null,
+    "fullDocumentation": null,
+    "fullName": "Waxholm Space atlas of the Sprague Dawley rat brain v4",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSD_atlas_v4.jsonld]: delete version id

File: instances/atlas/commonCoordinateSpace/WHSSD_v1-01.jsonld [hunk 17]
```diff
@@ -0,0 +1,23 @@
...
+    "@type": "https://openminds.ebrains.eu/sands/CommonCoordinateSpace",
+    "anatomicalAxesOrientation": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/anatomicalAxesOrientation/RAS"
+        }
+    ],
+    "axesOrigin": null,
+    "defaultImage": null,
+    "digitalIdentifier": null,
+    "fullName": "Waxholm Space of the Sprague Dawley rat brain - Coordinate system v1.01",
```
Comment (lzehl) [inline instances/atlas/commonCoordinateSpace/WHSSD_v1-01.jsonld]: "Waxholm Space of the Sprague Dawley rat brain (coordinate space)"

File: instances/atlas/commonCoordinateSpace/WHSSD_v1-01.jsonld [hunk 18]
```diff
@@ -0,0 +1,23 @@
...
+    "axesOrigin": null,
+    "defaultImage": null,
+    "digitalIdentifier": null,
+    "fullName": "Waxholm Space of the Sprague Dawley rat brain - Coordinate system v1.01", 
+    "homepage": null,
+    "howToCite": null,
+    "nativeUnit": null,
+    "ontologyIdentifier": null,
+    "releaseDate": "2014-07-16",
+    "shortName": "Waxholm space v1.01 (rat)",
```
Comment (lzehl) [inline instances/atlas/commonCoordinateSpace/WHSSD_v1-01.jsonld]: "WHSSD v1.01" ?

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/atlas/brainAtlas/WHSSD_atlas.jsonld (hunk 1 above)
Comment (UlrikeS91) [reply instances/atlas/brainAtlas/WHSSD_atlas.jsonld]: Compromise: WHSSD rat brain atlas

File: instances/atlas/brainAtlasVersion/WHSSD_atlas_v1-01.jsonld (hunk 7 above)
Comment (UlrikeS91) [reply instances/atlas/brainAtlasVersion/WHSSD_atlas_v1-01.jsonld]: done

File: instances/atlas/commonCoordinateSpace/WHSSD_v1-01.jsonld (hunk 17 above)
Comment (UlrikeS91) [reply instances/atlas/commonCoordinateSpace/WHSSD_v1-01.jsonld]: done

File: instances/atlas/commonCoordinateSpace/WHSSD_v1-01.jsonld (hunk 18 above)
Comment (UlrikeS91) [reply instances/atlas/commonCoordinateSpace/WHSSD_v1-01.jsonld]: ok...

**Comment by UlrikeS91** [comment]
All comments/suggestions integrated and noticed that I forgot "isNewVersionOf". So, fixed that too....

**Review by lzehl — APPROVED** [review]

**Pull request merged**